Dockerfile

```diff
@@ -34,8 +36,8 @@ RUN chown -R app /app
 USER app
 
 COPY Gemfile Gemfile.lock /app/
-RUN gem install bundler
-RUN bundle install --frozen --path vendor/bundle
+RUN gem install bundler:2.0.2
```

Comment:
Ditto, as per my yarn comment. The Gemfile.lock could specify a new version of bundler and the install might fail. Is there away to pick the bundler version from the lockfile?

Reply:
This gives us a kind of chicken and egg situation, where we need bundler to install from the lock file. If bundler fails, we should issue a PR to the Dockerfile to fix it?